### DISCUSSION TITLE
Use schema-registry with Strimzi Kafka Cluster

### DISCUSSION THREAD

**Discussion by murthy95** [strimzi/strimzi-kafka-operator#7396] [Q&A]
What's the best way to deploy schema registry on k8s to run with Strimzi Kafka cluster?

I tried using StrimziSchemaRegistry resource from [here](https://github.com/lsst-sqre/strimzi-registry-operator) but ran into issues that I raised [here](https://github.com/lsst-sqre/strimzi-registry-operator/issues/77)

**Comment by scholzj** [comment]
Strimzi does not have any special support for schema registries. Schema Registries are applications as any other - some of them act as Kafka clients and use Kafka as a storage, some of them don't connect to Kafka at all and use other storage options. But in general,  the deployment does not differ from any other application.
